1_fetch/src/get_nwis_data.R

```diff
@@ -1,35 +1,19 @@
+#' Getting NWIS Data 
+#'
+#' @param site_num character, 4 different site_num: "01427207", "01432160", "01436690", "01466500"
```

Comment:
I love that you're documenting the functions like this. Here you could say "a site number for a USGS streamgage" instead of listing the specific sites being used, since this function will work with any site.